api/app/src/TestHelpers/ClientTestHelper.php

```diff
@@ -11,21 +11,24 @@
 use Doctrine\ORM\EntityManager;
 use Faker\Factory;
 use PHPUnit\Framework\TestCase;
+use Prophecy\Prophet;
```

Comment:
Can we get rid of prophet from the unit tests? I'd prefer to use vanilla phpunit for new code.